src/util/link-helper.ts

```diff
@@ -63,3 +63,27 @@ function isRelativeLink(input: string) {
 function hasRelativeProtocol(input: string) {
     return input.startsWith('//');
 }
+
+/**
+ * Returns the appropriate `rel` attribute value for a link.
+ *
+ * If an explicit `rel` value is provided, it will be used.
+ * Otherwise, when `target` is `_blank`, automatically returns
+ * `"noopener noreferrer"` for improved security.
+ *
+ * @param target - The target attribute value (e.g., "_blank", "_self")
+ * @param explicitRel - An explicitly provided rel attribute value
+ * @returns The rel attribute value to use, or undefined if none needed
+ */
+export function getRel(target?: string, explicitRel?: string) {
+    if (explicitRel !== undefined) {
+        return explicitRel.trim() || undefined;
+    }
+
+    const normalizedTarget = target?.toLowerCase();
+    if (normalizedTarget === '_blank') {
```

Comment:
Very much a nitpick: when in doubt, it's definitely better to use a named variable, like here, but in this case I feel like it's just a tiny bit on the overzealous side. Keep it or leave it, it's up to you. Here's a committable suggestion if you want it 🙂

```suggestion
    if (target?.toLowerCase() === '_blank') {
```

```
fixup! chore(link-helper): add a utility function automatically setting `rel`
```